apps/design-system/src/design-tokens/Borders.stories.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta } from '@storybook/blocks'
+
+<Meta 
+    title="Design Tokens/Borders" 
+/>
+
+# Borders
+
+## Border width
+
+|Token|Description|
+|:--|:--|
+|`border-0`|No border|
+|`border-2`|2px border|
+|`border-4`|4px border|
+|`border-8`|8px border|
+
+For more variations, see https://tailwindcss.com/docs/border-width.
+
+## Border style
+
+|Token|Description|
+|:--|:--|
+|`border-solid`|(Default) Solid border|
+|`border-dashed`|Dashed border|
+|`border-dotted`|Dotted border|
+|`border-none`|No border|
+
+For more variations, see https://tailwindcss.com/docs/border-style.
+
+## Border radius
+
+|Token|Description|
+|:--|:--|
+|`rounded-none`|(Default) No rounded border|
+|`rounded-sm`|2px radius|
+|`rounded`|4px radius|
+|`rounded-md`|6px radius|
+|`rounded-lg`|8px radius|
```

Comment:
just curious, In some cases in the figma design, we have components with radios of 9px or spacing of 5px, should they follow the tokens or a new token be created for those spacing when need arise?

Reply:
I should either update the designs or we can create extra classes for these cases, we'll see. In any case, this is like a styleguide, they should not be added here just because one or two components have to use these styles. You see what I mean?

Reply:
Got it. thanks.